app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/ReviewGoodsController.scala

```diff
@@ -36,21 +36,21 @@ class ReviewGoodsController @Inject()(override val controllerComponents: Message
 
   val onPageLoad: Action[AnyContent] = actionProvider.journeyAction { implicit request =>
     request.declarationJourney.goodsEntries.map(Goods.apply) match {
-      case Seq() => Redirect(routes.SkeletonJourneyController.searchGoods())
+      case Seq() => Redirect(routes.SearchGoodsController.onPageLoad())
```

Comment:
maybe /invalid-request ?

Reply:
I'm not actually sure how this will behave. Goods.apply throws a RuntimeException which I think would mean if there is an incomplete GoodsEntry it will throw a 500 rather than this map returning a Seq.empty and this case being matched. I think this case being matched relies on declarationJourney.goodsEntries being a Seq.empty

Reply:
Yeah I think goodsEntries.map(Goods.apply) is probably the wrong method because it will give an internal server error.

I think it should redirect somewhere consistent.

The question is whether we redirect in all such cases to /search-goods, or to /invalid-request. 

Nick has said we should always go to an exception page, so I think we should go to /invalid-request whenever the goods entry is in an invalid state.

I'd probably add a test to show it working.